### DISCUSSION TITLE
How to disable table like rendering of gist?

### DISCUSSION THREAD

**Discussion by aakashnand** [nunocoracao/blowfish#1219] [Q&A]
I am using GitHub gist and it's rendering like the following.

<img width="727" alt="image" src="https://github.com/nunocoracao/blowfish/assets/14219201/2201a0ab-2a73-4984-9d51-93b454fa4e0e">

How do display the gist like this without lines in between?

<img width="739" alt="image" src="https://github.com/nunocoracao/blowfish/assets/14219201/55126547-e648-4c55-9976-c0412105f3db">

When inspecting through inspect tools in the browser, I found out that when I disable `border-style: solid` it removes the lines.
and this style is coming from

https://github.com/nunocoracao/blowfish/blob/ddf79f4573edfdd40592fad843cab508b1885cfb/assets/css/compiled/main.css#L31

Is there any way to disable the border for the GitHub gist?

**Comment by seqizz** [comment]
Same thing also disturbed me, and since we can override theme's css with our custom, adding a file named `assets/css/custom.css` with following in it solves this:
```
.gist .highlight-wrapper tr {
    border-style: none;
}
```
